generator/explores/glean_ping_explore.py

```diff
@@ -10,9 +15,36 @@ class GleanPingExplore(PingExplore):
 
     type: str = "glean_ping_explore"
 
+    def _to_lookml(self, v1_name: Optional[str]) -> List[Dict[str, Any]]:
+        """Generate LookML to represent this explore."""
+        repo = next((r for r in GleanPing.get_repos() if r["name"] == v1_name))
+        glean_app = GleanPing(repo)
+        # convert ping description indexes to snake case, as we already have
+        # for the explore name
+        ping_descriptions = {
+            k.replace("-", "_"): v for k, v in glean_app.get_ping_descriptions().items()
+        }
+        # collapse whitespace in the description so the lookml looks a little better
+        ping_description = " ".join(ping_descriptions[self.name].split())
```

Comment:
I added this because the lookml looked bad with all the newlines and spaces. Should we do this in more places?

Reply:
I think descriptions are the only place where this might happen. SQL statements we want to keep the whitespace as-is so it's more readable in the LookML files. We could add this logic to Glean Metric descriptions as well.

Reply:
I can look into doing this in a followup